docs/sources/tutorials/alerting-get-started/index.md

```diff
@@ -103,9 +146,14 @@ Next, let's configure a contact point in Grafana's Alerting UI to send notificat
 
 We have created a dummy Webhook endpoint and created a new Alerting contact point in Grafana. Now, we can create an alert rule and link it to this new integration.
 
+<!-- INTERACTIVE page step2.md END -->
+
+<!-- INTERACTIVE page step3.md START -->
+
+
 ## Create an alert
 
-Next, we'll establish an [alert rule](http://grafana.com/docs/grafana/next/alerting/fundamentals/alert-rule-evaluation/) within Grafana Alerting to notify us whenever our sample app experiences a specific volume of requests.
+Next, we'll establish an [alert rule](http://grafana.com/docs/grafana/next/alerting/fundamentals/alert-rule-evaluation/) within Grafana Alerting to notify us whenever alert rules are triggered and resolved.
```

Comment:
(Sorry, this is awkward, GitHub only lets you comment on changed lines....)

Line 158 should be the first step in the procedure.
Line 160 doesn't need to be a heading.
Line 162 should be the second step, "Enter a name for your alert rule.  Make it short.....etc."